packages/nc-gui-v2/components/virtual-cell/Lookup.vue

```diff
@@ -11,7 +11,7 @@ provide(ReadonlyInj, true)
 const column = inject(ColumnInj) as ColumnType & { colOptions: LookupType }
 const meta = inject(MetaInj)
 const value = inject(ValueInj)
```

Comment:
Can we possibly rename `ValueInj` to something like `CellValueInj`?
So it doesn't end up with `value.value`

Reply:
agree i will update that :+1: